### ISSUE TITLE
Browsersync localhost:3000 doesnt work

### ISSUE THREAD

**Issue by justlevine** [ahmadawais/WPGulp#110]
### Expected Behavior

Visiting `localhost:3000` should load the BrowserSync-ed version of the website.

### Actual Behavior

Visiting localhost:3000 gives you an ERR_CONNECTION_CLOSED.
Visiting the **External Url** works as expected.

### Steps to Reproduce

1.  Edit your `wpgulp.config.js` project url.
2. (Optional) set `browserAutoOpen: true`
3. type `npm start`
4.  If you didn't set browserAutoOpen to true, then visit localhost:3000

1. `node -v`: 8.11.1
2. `npm -v`: 5.6.0
3. `browser-sync`: @2.24.7

Then, specify:

1. Operating system: Windows 10
2. Browser and version (if relevant): Chrome v68

**Possible Solution**

Edit gulpfile.babel.js in line 86 `browserSync.init`:
`proxy: config.projectURL` => `host: config.projectURL`
and add `proxy: 'http://yoursite.test'`

**Comment by ahmadawais** [comment]
Is it still the same since the last two versions? I mean try now, WPGulp has been updated.

**Comment by tnchuntic** [comment]
I am experiencing this too :( 

No code changes on my part just testing the WPGulp.

Its a bit weird because of I can access port 3001 for the browser sync UI but for the dev website its not working.
UI:
https://snag.gy/OplCrA.jpg
Dev site:
https://snag.gy/hBK1Sv.jpg

Terminal
https://snag.gy/zpIAtr.jpg

**Comment by commwork** [comment]
I had this problem when using another terminal, with CMD or Pwershell no problem (Windows 10).

**Comment by ashbryant** [comment]
Yep, I'm having issues with this also (mac) I can view :3001 fine but nothing else

**Comment by RyanCadby** [comment]
I am having the same issue, wpgulp works on my windows pc but does not work on my mac. Everything seems to compile fine but the browser just refreshes endlessly... I have cleared my dns cache and changed my host file. nothing seems to work

**Comment by elharony** [comment]
This issue still exists. It keeps refreshing endlessly, just like @RyanCadby

**Comment by elharony** [comment]
For anyone still facing this issue, it's because the `projectURL` wrong value.

How do you access your WordPress Project? Let's say it's inside `xampp/htdocs/projects/new-wordpress-site/` and in order to access it using localhost, you type: `localhost/projects/new-wordpress-site/`.

Copy that path, and give it to `projectURL` so it can access your Project Files.

❌ `projectURL: 'localhost:3000'`
❌ `projectURL: 'localhost:3001'`
❌ `projectURL: 'localhost'`
❌ `projectURL: '127.0.0.1'`
✔️ `projectURL: 'http://localhost/projects/new-wordpress-site/'`
